client/src/components/NotesList/NotesList.jsx

```diff
@@ -1,12 +1,24 @@
 import style from "./NotesList.module.css";
 
-export function NotesList({ noteListData }) {
+export function NotesList({
+  noteListData,
+  handleCreateNewNote,
+  handleNoteSelection,
+}) {
+  console.log(noteListData);
   return (
     <div className={style.notesList}>
       <ul>
+        <li className={style.noteElement} onClick={handleCreateNewNote}>
+          Create a New Note
+        </li>
         {noteListData.map((note, index) => {
           return (
-            <li key={index}>
```

Comment:
Just curious is there any particular reason you used index to set the key instead of not.id ?